docs/source/tpu.rst

```diff
@@ -192,7 +192,58 @@ set the 16-bit flag.
 
 Under the hood the xla library will use the `bfloat16 type <https://en.wikipedia.org/wiki/Bfloat16_floating-point_format>`_.
 
-----------------
+
+-----------------
+
+Weight Sharing/Tying
+-----------------------
```

Comment:
```suggestion
Weight Sharing/Tying
--------------------
```
nice maybe link this to the `Performance considerations` section Adrian created recently. Seems important.

Reply:
Not sure we can group this under `performance consideration`. This is more of an item that needs to be kept a note of i guess. Anyways, this section will follow the `performance considerations` section itself.

Reply:
yeah, I was suggesting to keep the note... just link the note section in those points.. anyways if it follows that one, then it's fine.